app/src/renderer/vuex/getters.js

```diff
@@ -83,7 +83,7 @@ export const depositDenom = getters =>
   getters.governanceParameters.loaded &&
   getters.governanceParameters.parameters.deposit.min_deposit
     ? getters.governanceParameters.parameters.deposit.min_deposit[0].denom
-    : ``
+    : `token`
```

Comment:
...?

Reply:
only a fallback value as long as the actual denom is not loaded.

Reply:
so if we use something like: "You can use your `denomDeposit`s to vote". It would read better for the user while loading.

Reply:
To avoid what happen here:
https://files.slack.com/files-pri/T564PPNET-FGC3GBTB6/image.png